tasks/ceph_deploy.py

```diff
@@ -24,11 +24,47 @@ def download_ceph_deploy(ctx, config):
     """
     Downloads ceph-deploy from the ceph.com git mirror and (by default)
     switches to the master branch. If the `ceph-deploy-branch` is specified, it
-    will use that instead.
+    will use that instead. The `bootstrap` script is ran, with the argument
+    obtained from `python_version`, if specified.
     """
+    ceph_admin = ctx.cluster.only(teuthology.get_first_mon(ctx, config))
+
+    try:
+        py_ver = str(config['python_version'])
+    except KeyError:
+        pass
+    else:
+        supported_versions = ['2', '3']
+        if py_ver not in supported_versions:
+            raise ValueError("python_version must be: {}, not {}".format(
+                ' or '.join(supported_versions), py_ver
+            ))
+
+        log.info("Installing Python")
+        for admin in ceph_admin.remotes:
+            system_type = teuthology.get_system_type(admin)
+
+        if system_type == 'rpm':
+            if py_ver == '3':
+                ctx.cluster.run(args=[
+                    'sudo', 'yum', '-y', 'install', 'epel-release',
+                    run.Raw('&&'),
+                    'sudo', 'yum', '-y', 'install',
+                    'python-virtualenv', 'python34'
```

Comment:
How can I check whether Python 3 was installed or it was already there?
We have to be absolutely sure, because Python 3 is an integral part of some distros.